File: sign/sign-interface/src/main/java/cn/moonnow/sign/vo/SessionVO.java
package cn.moonnow.sign.vo;

import java.io.Serializable;

import lombok.Data;

/**
 * 会话 DEV_SESSION 展示类
 */
@Data
public class SessionVO implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String sessionId; // 会话编号

  protected String accountId; // 账号编号

  protected String loginKey; // 登录标识

  protected Long currentTimes; // 当前时间

  protected String aesKey; // AES密钥

}

File: vue-dev-tool/src/view/cn/moonnow/sign/session/SessionVO.js
export default class SessionVO {
  constructor () {
    this.sessionId = null // 会话编号
    this.accountId = null // 账号编号
    this.loginKey = null // 登录标识
    this.currentTimes = null // 当前时间
    this.aesKey = null // AES密钥
  }
}
